templates/portfolios/detail.html

```diff
@@ -28,15 +28,19 @@
 % block body-wide %
 {% block body %}
 
-<div class="portfolio-top">
-  <div class="container">
+    <div class="portfolio-top">
+    <div class="container">
     <div class="portfolio-main">
-      <a id="new-project" onclick="show_select_portfolio_modal(); return false;" href="#"" class=" btn
+        <a id="new-project" onclick="show_select_portfolio_modal(); return false;" href="#" class=" btn
         btn-success">Start a project</a>
-      {% if can_invite_to_portfolio %}
-      <a href="#" onclick="invite_user_into_portfolio(); return false;" id="grant-portfolio-access"
-        class="btn btn-success"><i class="glyphicon glyphicon-plus"></i> Grant access to member</a>
-      {% endif %}
+        {% if can_change_portfolio %}
+            <a id="manage-portfolio" onclick="show_manage_portfolio_modal('{{portfolio.id}}'); return false;" href="#" class=" btn
+            btn-success">Manage Portfolio</a>
```

Comment:
This is definitely a nit-pick, but it seems odd to split the line here. If the line is too long, maybe split at the start of the "class" attribute so it's clear what all the classes here. But again, very small nitpick.